packages/prop-house-webapp/src/components/HouseHeader/index.tsx

```diff
@@ -62,44 +87,19 @@ const HouseHeader: React.FC<{
           </div>
 
           <div className={classes.propHouseDataRow}>
-            <div className={classes.itemData}>{community.numAuctions ?? 0}</div>
```

Comment:
so we still need this nullish check because if not we get a blank here versus a literal 0. 

referenced in PR https://github.com/Prop-House/prop-house-monorepo/pull/313

<img width="361" alt="Screenshot 2022-11-22 at 4 19 21 PM" src="https://user-images.githubusercontent.com/26611339/203423223-170723b3-b521-4a34-aa5e-b3ae48a33cb6.png">